.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: objective-c
+osx_image: xcode7
+
+env:
+  matrix:
+    - LC_CTYPE=en_US.UTF-8
```

Comment:
This does not create any build matrix, so this should be in the following `global` section?